step-templates/sentry-release-tracking-token-auth.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Id": "b40d5528-5de0-422d-a746-1e93cd04dea0",
+  "Name": "Sentry Release Tracking (Token Auth)",
+  "Description": "Posts a new release to Sentry, It can optionaly resolve all previous issues.\n\nIf the release already exists, it only applies the resolving.",
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptBody": "$url = \"https://sentry.io/api/0/organizations/$organization/releases/\"\nWrite-Host $url\n\n$headers = New-Object \"System.Collections.Generic.Dictionary[[String],[String]]\"\n$headers.Add(\"Authorization\", \"Bearer $sentryApiKey\")\n$body = ConvertTo-Json @{ \n\t\"version\" = $OctopusParameters['Octopus.Release.Number']\n    \"projects\" = $projects.Split(\";\")\n}\n\nWrite-Host $body\nTry\n{\n\t$response = Invoke-RestMethod -Method Post -Uri \"$url\" -Body $body -Headers $headers -ContentType \"application/json\"\n\tWrite-Host $response\n}\nCatch [System.Net.WebException] \n{\n\tWrite-Host $_\n\tif($_.Exception.Response.StatusCode.Value__ -ne 400)\n\t{\n\t\tThrow\n\t}\n}\n",
```

Comment:
I checked out the Script body, and generally looks fine. I noticed that the error handling would only _reliably_  work for `Windows PowerShell`.

```powershell
Try
{
	$response = Invoke-RestMethod -Method Post -Uri "$url" -Body $body -Headers $headers -ContentType "application/json"
	Write-Host $response
}
Catch [System.Net.WebException] 
{
	Write-Host $_
	if($_.Exception.Response.StatusCode.Value__ -ne 400)
	{
		Throw
	}
}
```

The `Catch [System.Net.WebException]` wouldn't fire for `PowerShell Core (pwsh)` since an exception surfaces as a  `Microsoft.PowerShell.Commands.HttpResponseException`. 

That's not an issue in itself, but if this step template i designed to _only_ works on Windows, then a note in the description would be helpful :+1: 

If not and it should work on both Linux/Windows, then I think you can simplify your script to this (please test this!):

```powershell
Try
{
	$response = Invoke-RestMethod -Method Post -Uri "$url" -Body $body -Headers $headers -ContentType "application/json"
	Write-Host $response
}
Catch 
{	
	if($_.Exception.Response.StatusCode -ne 400)
	{
		Throw $_
	}
}
```